shared/status.js

```diff
@@ -35,7 +43,9 @@ const getStatus = () => {
 			const engines = enginesArg === 'all' ?
 				['chakra', 'javascriptcore', 'spidermonkey', 'v8', 'xs'] :
 				enginesArg.split(',');
-			status.engines = engines;
+			status.engines = engines.map(engine => translate[engine] ?
+				translate[engine] :
+				engine);
```

Comment:
The `map` call is redundant in the case of `enginesArg === 'all'`.

Reply:
sure, I can move .map to chain into the `enginesArg.split(',')` part.